shvutil.c

```diff
@@ -138,7 +138,7 @@ ShUtilvAllocateContiguousMemory(
 	// If we're here then we'll just fall back to using executable
 	// memory.
 	//
-#pragma warning(suppress: 30029)
+#pragma warning(suppress: 30030)
```

Comment:
Sorry, didn't mean to PR these changes (still getting used to the github interface), but thanks for the comment.

I can't find that header, and can you give an example of using the comment feature?  I don't see that listed at https://msdn.microsoft.com/en-us/library/2c8f766e.aspx